Sources/TensorFlow/Loss.swift

```diff
@@ -109,7 +109,7 @@ public func hingeLoss<Scalar: TensorFlowFloatingPoint>(
 ///   - expected: Expected values, i.e. targets, that correspond to the correct output.
 @differentiable(wrt: (predicted, expected))
 public func cosineSimilarity<Scalar: TensorFlowFloatingPoint>(
-    predicted: Tensor<Scalar>, expected: Tensor<Scalar>
+    _ predicted: Tensor<Scalar>, _ expected: Tensor<Scalar>
```

Comment:
Thanks for the change. With this, the call sites no longer need to write `predicted:` or `expected:`, which is good. However, Since this is an API for more general use than loss functions, we should not distinguish between `predicted` and `expected` at the implementation site or in API documentation comments. As such, could you please rename argument names `predicted` and `expected` to `x` and `y` respectively?

Reply:
@rxwei How about I put this as a Math op and have a higher level function within losses calling the base function? would that count as redundancy of code? Because It's just sort of weird that we are putting a general API usage function in losses.swift

Reply:
Putting it in Math.swift is good. A wrapper in Loss.swift is unnecessary.